modules/installing-rhv-insecure-mode.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-custom.adoc
+// * installing/installing_rhv/installing-rhv-default.adoc
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installing-rhv-insecure-mode_{context}"]
+= Installing {product-title} on {rh-virtualization} in insecure mode
+
+By default, the installer creates a valid CA certificate, prompts you for confirmation, and stores the certificate to use during installation. You do not need to create or install one manually.
```

Comment:
I would replace "valid CA certificate" with "certificate, that is more accurate.

Reply:
@janoszen Done.